config/jobs/kubernetes/sig-release/kubernetes-builds.yaml

```diff
@@ -165,23 +133,23 @@ periodics:
           memory: "8Gi"
 
 # manual-release-bump-required
-- interval: 1h
+- interval: 30m
   name: ci-kubernetes-build-stable2
   labels:
     preset-service-account: "true"
     preset-dind-enabled: "true"
   spec:
     containers:
-    - image: gcr.io/k8s-testimages/bootstrap:v20190211-fab6034b4
+    - image: gcr.io/k8s-testimages/bootstrap:v20190213-2f4f06285
```

Comment:
should not we version the bootstrap image per release?

Reply:
We version kubekins per release because different release want to use different go/bazel versions. This job doesn't require any of those so version shouldn't matter.